packages/syft/src/syft/custom_worker/config.py

```diff
@@ -0,0 +1,88 @@
+# stdlib
+from hashlib import sha256
+from pathlib import Path
+from typing import List
+
+# third party
+from packaging import version
+from pydantic import validator
+from typing_extensions import Self
+import yaml
```

Comment:
We should add PyYAML to syft dependencies as well? PyYAML is currently listed as dependency in hagrid and syft-cli, not syft.

Reply:
yes agreed, lets add that to the syft setup.cfg